CMakeLists.txt

```diff
@@ -1,15 +1,19 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
 
-PROJECT (xcdf CXX C)
+CMAKE_POLICY(SET CMP0048 OLD) # Don't tet the project() command manage VERSION variables
```

Comment:
It's usually much better to switch to the NEW behavior

Reply:
I decided to keep the old one because I didn't want to modify the current code at
https://github.com/jimbraun/XCDF/blob/176628a1373ae6c21d6cff9ed139676ed527a442/CMakeLists.txt#L39
which could be used in more places elsewhere